app/controllers/pages/date_settings_controller.rb

```diff
@@ -0,0 +1,50 @@
+class Pages::DateSettingsController < PagesController
+  def new
+    input_type = session.dig(:page, "answer_settings", "input_type")
+    @date_settings_form = Forms::DateSettingsForm.new(input_type:)
+    @date_settings_path = date_settings_create_path(@form)
+    @back_link_url = type_of_answer_new_path(@form)
+    render "pages/date_settings"
+  end
+
+  def create
+    @date_settings_form = Forms::DateSettingsForm.new(date_settings_form_params)
+    @date_settings_path = date_settings_create_path(@form)
+    @back_link_url = type_of_answer_new_path(@form)
+
+    if @date_settings_form.submit(session)
+      redirect_to new_page_path(@form)
+    else
+      render "pages/date_settings"
+    end
+  end
+
+  def edit
+    @page = Page.find(params[:page_id], params: { form_id: @form.id })
+    input_type = @page&.answer_settings&.input_type
```

Comment:
Really don't like our usage of `&` to fail gracefully. We should try and make sure all questions have at least answer_settings set

Reply:
Good point - have made a placeholder card for this so we don't forget: https://trello.com/c/RDzeLJqz/425-write-migrations-for-changed-autofill-fields